### ISSUE TITLE
Clarification

### ISSUE THREAD

**Issue by crisdosaygo** [crislin2046/browsercontrollers.store#1]
There should be 1 browser controller that opens and shuts chrome and manages the devtools port.

This controller can register plugins or whatever they're called.

Alternately they could be the controllers and the manager is the orcehstrator.

In any case there should not be multiple binaries all handling chrome.

There should be one. And the browserctontroller boilerplate is for the plugins to that system.

Basically, it's like go language. You install a plugin by pulling it from source, and running it's node.

You get a little entry in the "plugins" page or whatever where you can switch it off, remove it, whatever.

That's a great idea I think.
